infermedica_api/webservice.py

```diff
@@ -479,6 +479,69 @@ def risk_factors_list(self):
 
         return models.RiskFactorList.from_json(response)
 
+    def red_flags(self, diagnosis_request, interview_id=None, max_results=8):
```

Comment:
Can you swap interview_id with max_results ?
So it will be aligned with the definition of suggest method `def suggest(self, diagnosis_request, max_results=8, interview_id=None):`
Those two endpoint from technical/structure point of view are the same.

Reply:
Sure, no problem!